apps/atqr/web/src/pages/termsAndCondition/termsConditions.pug

```diff
@@ -0,0 +1,92 @@
+html
+  head
+    meta(charset='utf-8')
+    meta(http-equiv='x-ua-compatible', content='ie=edge')
+    title ATQR - AJOELHOU TEM QUE REZAR
+    meta(name='robots', content='noindex, follow')
+    meta(name='description', content='')
+    meta(name='viewport', content='width=device-width, initial-scale=1, shrink-to-fit=no')
+
+    //- Favicon
+    link(rel='shortcut icon', type='image/x-icon', href='../../assets/images/favicon.png')
+
+    //- Google Fonts
+    link(rel='preconnect', href='https://fonts.googleapis.com')
+    link(rel='preconnect', href='https://fonts.gstatic.com', crossorigin)
+    link(
+      href='https://fonts.googleapis.com/css2?family=Big+Shoulders+Display:wght@300;400;500;600;700;800&family=Open+Sans:wght@300;400;500;600;700;800&display=swap',
+      rel='stylesheet'
+    )
+    link(
+      href='https://fonts.googleapis.com/css?family=Archivo:wght@300;400;500;600;700&display=swap',
+      rel='stylesheet'
+    )
+
+    //- CSS
+    link(rel='stylesheet', href='../../assets/css/aos.css')
+    link(rel='stylesheet', href='../../assets/css/flaticon.css')
+    link(rel='stylesheet', href='../../assets/fonts/fa-solid-900.woff')
+    link(rel='stylesheet', href='../../assets/css/style.css')
+    link(rel='stylesheet', href='../../assets/css/style.min.css')
+
+    link(rel='stylesheet', href='./terms-style.css')
+
+  body
+    mixin terms(title, paragraph)
+      h4.h4= title
```

Comment:
@Saulo, im in a review meeting with the team and could use your input on the use of a class with the same name as the tag.